docs/tutorials/ui-customization.md

```diff
@@ -91,7 +91,7 @@ ui.configure(config);
 An important note: the 'overflow_menu' button needs to be part of the 'controlPanelElements'
 layout for the overflow menu to be available to the user.
 
-The presence of the seek bar and the big play button in the center of the video element can be
+The presence of the seek bar and the big play button in the centre of the video element can be
```

Comment:
"centre" is the British english spelling, and "center" is the American english spelling. I'd keep it as "center" for consistency, since we use that elsewhere... in a number of places we can't change, too (e.g. the names of CSS property values).

Reply:
Understood!